### PR TITLE
Add GCP (and more) to cloudTypes parameter desc

### PR DESCRIPTION
**Pull request by ritawatson**
This PR fixes a doc bug (Jira below) to add GCP to the **cloudTypes** parameter description in the table. It also fixes the Validation head level in the Helms Registries doc (heading 2 to heading 1).

Jira: https://spectrocloud.atlassian.net/browse/DOC-425

[Preview](https://deploy-preview-1072--docs-spectrocloud.netlify.app/registries-and-packs/add-custom-packs)

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1072#" title="Author self-approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
@ritawatson: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | ad3af8bea1f6fe201ec5e5de6a6d157d4d57bdb2 | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1072/librarium_build/1608866458870222848) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: content/docs/09-registries-and-packs/3-add-custom-packs.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ Each pack contains a metadata file named `pack.json`. The table below explains i
 | layer | String | True | Relevant layer that this pack should be part of; such as os, k8s, cni, csi, addon |
 | addonType | String | False | Addon-type must be set for packs that have the layer set to Addon. The value must be one of the following: logging, monitoring, load balancer, authentication, ingress, security. Setting a relevant correct addon type ensures packs are organized correctly on the management console making it easy for profile authors to find packs. |
 | version | String | True | A Semantic version for the pack. It is recommended that the pack version be the same as the underlying integration it is being created for. For example, the version for the pack that will install Prometheus 2.3.4, should set to 2.3.4. |
-| cloudTypes | Array | True | Supported cloud types are AWS, Azure, VMware. One or more types can be provided for a pack. |
+| cloudTypes | Array | True | Supported cloud types are AWS, Azure, GCP, and VMware. One or more types can be provided for a pack. |
```
Comment (resmaa) [inline content/docs/09-registries-and-packs/3-add-custom-packs.md:38]: we need to add  openstack, maas and Tencent

**Review by resmaa — CHANGES_REQUESTED** [review]

File: content/docs/09-registries-and-packs/3-add-custom-packs.md (hunk 1 above)
Comment (liltigermel) [reply content/docs/09-registries-and-packs/3-add-custom-packs.md:38]: Actually here is the full list.  Sorry its a bit confusing with the description first and actual property value second

AWS - aws
AWS EKS - eks
Google - gcp
Azure - azure
Azure Managed Kubernetes Service- aks
VMware -vpehere
OpenStack - openstack
Tencent TKE -tke
MaaS -maas
Virtual Edge and virtual clusters - libvirt
Edge 1.0 containerized edge - edge
Edge 2.0 - edge-native 
CoxEdge -coxedge
Any of the supported cloud types -all

Comment (liltigermel) [reply content/docs/09-registries-and-packs/3-add-custom-packs.md:38]: Ideally we should say supported values are then have a table with the valid property values and the descriptions.  I am assuming we are a case sensitive platform so AWS isn't the same as AWS. Also today it says VMware, but the actually property value you need to specify is vsphere for example

Comment (ritawatson) [reply content/docs/09-registries-and-packs/3-add-custom-packs.md]: @liltigermel  
Added the above values to the cloudType description.

Comment (ritawatson) [reply content/docs/09-registries-and-packs/3-add-custom-packs.md]: @liltigermel Added the above values to cloudType description.

**Review by liltigermel — APPROVED** [review]
Approving.  The list of property values documented now matches what is in the code

**Review by karl-cardenas-coding — APPROVED** [review]
Minor suggestions, otherwise looks good.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1072#pullrequestreview-1237809510" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1072#pullrequestreview-1237784576" title="Approved">liltigermel</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1072#" title="Author self-approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-2/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1072#pullrequestreview-1237809510" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1072#pullrequestreview-1237784576" title="Approved">liltigermel</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1072#" title="Author self-approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/09-registries-and-packs/3-add-custom-packs.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ Each pack contains a metadata file named `pack.json`. The table below explains i
...
 | addonType | String | False | Addon-type must be set for packs that have the layer set to Addon. The value must be one of the following: logging, monitoring, load balancer, authentication, ingress, security. Setting a relevant correct addon type ensures packs are organized correctly on the management console making it easy for profile authors to find packs. |
 | version | String | True | A Semantic version for the pack. It is recommended that the pack version be the same as the underlying integration it is being created for. For example, the version for the pack that will install Prometheus 2.3.4, should set to 2.3.4. |
-| cloudTypes | Array | True | Supported cloud types are AWS, Azure, VMware. One or more types can be provided for a pack. |
+| cloudTypes | Array | True | You can provide one or more types for a pack. Supported values are as follows: <br /><br />**all**, **aws**, **azure**, **gcp**, **tencent**, **vsphere**, **openstack**, **baremetal**, **maas**, **aks**, **eks**,  **tke**, **edge**, **edge-native**, **coxedge**, and **libvirt** (virtual edge and virtual clusters).
```
Comment (resmaa) [inline content/docs/09-registries-and-packs/3-add-custom-packs.md:38]: "**libvirt** (virtual edge and virtual clusters)"

1. here libvirt is for virtualized edge
2.  we have a cloud type `edge` for a native edge.
3. for virtual clusters it's `all` as we deploy only add-ons in virtual clusters, infra layers are from the host cluster.

@brathina-spectro can you plz confirm the correctness of these observations?
cc: @karl-cardenas-coding

Comment (ritawatson) [reply content/docs/09-registries-and-packs/3-add-custom-packs.md]: @resmaa I checked with Boobalan, and he confirms that it should say 'libvirt (virtualized edge)'. I've made this change.

**Review by resmaa — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1072#pullrequestreview-1237809510" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1072#pullrequestreview-1237784576" title="Approved">liltigermel</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1072#pullrequestreview-1239649731" title="Approved">resmaa</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1072#" title="Author self-approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**